samwise.js

```diff
@@ -1,10 +1,10 @@
 /**
```

Comment:
SCRIPT-There seems to be a typo in the function name. Did you mean `calculateAverage` instead of `calculateAverge`?